bot/src/discord_commands/jibiki_all_search.js

```diff
@@ -0,0 +1,254 @@
+const {throwPublicErrorInfo} = require('./../common/util/errors.js');
+const {Navigation, NavigationChapter} = require('monochrome-bot');
+const axios = require('axios');
+const constants = require('./../common/constants.js');
+
+const jibikiApiUri = 'https://api.jibiki.app';
+const maxFieldsPerPage = 8;
+
+module.exports = {
+  commandAliases: ['jibiki', 'jb'],
+  cooldown: 3,
+  uniqueId: 'best_dictionary',
+  shortDescription: 'Search Jibiki.',
+  usageExample: '<prefix>jibiki 女らしい',
+  async action(bot, msg, suffix, monochrome) {
+    if (!suffix) {
+      const {prefix} = msg;
+      return throwPublicErrorInfo('jibiki', `Say **${prefix}jibiki [text]** to search for words. For example: **${prefix}jibiki dance**`, 'No suffix');
+    }
+
+    const response = await axios.get(`${jibikiApiUri}/all?query=${encodeURIComponent(suffix)}&sentenceCount=5`);
```

Comment:
Please also try-catch here, in case there is a timeout (which will get thrown as an error), and re-throw as a FulfillmentError, something like this:
```js
let response;
try {
  response = response = await axios.get(`${jibikiApiUri}/all?query=${encodeURIComponent(suffix)}&sentenceCount=5`);
} catch (err) {
  const errorEmbed = {
    embed: {
      title: 'Error',
      description: 'I timed out while trying to reach Jibiki, please try again later.',
      color: constants.EMBED_WRONG_COLOR,
    },
  };

  throw new FulfillmentError({
    publicMessage: errorEmbed,
    logDescription: 'Jibiki timeout',
    err,
  });
}
```
FulfillmentErrors are my way of handling and logging errors in a more controlled way in monochrome. The `publicMessage` gets sent to the user and the `logDescription` goes into my logs, and it will log as a warning instead of an error. Since timeouts happen sometimes and I don't need to be alerted (errors will alert me).